pod/perlvar.pod

```diff
@@ -1120,6 +1120,26 @@ information on this form and its uses.
 
 This variable was added in 5.25.7
 
+If you need access to this functionality in older Perl's you can use this
+function immediately after your regexp.
+
+    sub get_captures {
+        my $str      = $&;
+        my @captures = ();
+
+        # Build array using the offsets from @- and @+
+        for (my $i = 1; $i < @-; $i++) {
```

Comment:
Why use a C-style for loop here? This is a simple counting loop, so
```perl
for my $i (1 .. $#-) {
```
expresses the idea more directly (and slightly more efficient).

Reply:
Agree to disagree here... `$#-` is too much punctuation and not as easily understood.

I've been using Perl for 20+ years and I just learned about `#` syntax in arrays today.

Reply:
If you don't like `$#-`, `@- - 1` is the same thing. (Doesn't really look better with `@-`, though, IMHO).